tests/snappi_tests/files/helper.py

```diff
@@ -131,11 +133,15 @@ def save_config_and_reboot(node, results=None):
         reboot(node, localhost, reboot_type=reboot_type, safe_reboot=True)
         logger.info("Wait until the system is stable")
         wait_until(180, 20, 0, node.critical_services_fully_started)
+        wait_until(180, 20, 0, check_interface_status_of_up_ports, node)
 
     # Convert the list of duthosts into a list of tuples as required for parallel func.
     args = set((snappi_ports[0]['duthost'], snappi_ports[1]['duthost']))
     parallel_run(save_config_and_reboot, {}, {}, list(args), timeout=900)
 
+    # Wait for internal bgp to come up.
+    time.sleep(100)
```

Comment:
same comment to add bgp check instead of hard coded value.

Reply:
@sdszhang : Done.